pkg/file/file_test.go

```diff
@@ -44,33 +38,38 @@ const yamlFile = `
 // This is a yaml file
 `
 
-func CreateKeptnConfigServiceMock(t *testing.T) *keptnfake.MockConfigService {
+func CreateKeptnResourceServiceMock(t *testing.T) *keptnfake.MockResourceHandler {
 
 	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 
-	return keptnfake.NewMockConfigService(mockCtrl)
+	return keptnfake.NewMockResourceHandler(mockCtrl)
 }
 
-func TestMountFiles(t *testing.T) {
+/*func TestMountFiles(t *testing.T) {
```

Comment:
I think that we should still test the mounting of the files logic

Reply:
Sorry, these tests have been commented out while testing. I just added them again with the new mocks and dependencies.